### ISSUE TITLE
Fetch Full Installer doesn't work on Big Sur

### ISSUE THREAD

**Issue by matlanz** [grahampugh/erase-install#82]
The flag --fetch-full-installer doesn't work when running this script in Big Sur.

The logic in line 1049 expects os_minor_version to be greater than or equal to 15 (Catalina) but the code to get that version has a hardcoded "10" it replaces, so line 986 actually returns "11" (and not the actual minor version like 2 for 11.2.3). Either way 11 or 2 are both lesser than 15, so FFI is not attempted.

Line 1049: `if [[ $ffi && $os_minor_version -ge 15 ]]; then`

Line 986: `os_minor_version=$( echo "$os_version" | sed 's|^10\.||' | sed 's|\..*||' )`

As explained by Scripting OSX [here](https://scriptingosx.com/2020/09/macos-version-big-sur-update/) there are multiple ways to fix this. Simplest being just adding a 'os_major_version' and comparing that first.

```
majorVersion=$(sw_vers -productVersion | awk -F. '{ print $1; }')
minorVersion=$(sw_vers -productVersion | awk -F. '{ print $2; }')

if [[ $majorVersion -ge 11 || $minorVersion -ge 14 ]]; then
  echo "Mojave or higher"
fi
```

**Comment by grahampugh** [comment]
Hi @matlanz good find! I did fix that in other parts of the script, but hadn't updated the part with `--fetch-full-installer`. It should be fixed in 0.9.1 which I'll push as a pre-release in a few moments.

**Comment by matlanz** [comment]
@grahampugh Awesome, thank you!
